routes/channel.js

```diff
@@ -48,6 +51,25 @@ function getStatus(withTree, req, res) {
 	})
 }
 
+function getEvents(req, res, next){
+	const { uid } = req.session
+	const resp = { channel: req.channel }
+
+	const eventsCol = db.getMongo().collection('eventAggregates')
+	const key = `events.IMPRESSION.${uid}`
+
+	return eventsCol
+	.find({
+		[ key ]:  { "$exists": true }
+	})
+	.limit(cfg.EVENTS_FIND_LIMIT)
+	.toArray()
+	.then(function(events) {
+		res.send({ ...resp, events })
```

Comment:
also in the future this API should support aggregating the event aggregates by time periods; but let's leave this out of the PR

Reply:
okay then